### ISSUE TITLE
A11y_AzureDataStudio_BigDataClusterOverviewPage_ScreenReader :Missing tab structure for the controls under the "Cluster Details".

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7817]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#DesktopApp;#VoiceOver;#SQL Azure Data Studio;#Mac

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS:macOS Mojave Version 10.14.6(18G87)

**Additional Details:**
Tool Name : VoiceOver

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
8. "Big Data Cluster Overview page " should open.
9. Start VoiceOver.
10. Navigate till "Cluster Details " using tab .
11. Observe the announcement by screen reader .

**Actual:**
The "SQL Servers ","HDFS", "Sparks", "Control", "Gateway" and "App" controls which are visually appearing as tab structure but are not structured as tabs.
Similar for "Basic data cluster Overview"
![image](https://user-images.githubusercontent.com/54656883/66652904-6f496600-ec54-11e9-8fd3-0b34b732fece.png)

**Similar issue is also observed with the following screens:**
1)Control tab:
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard->click on the "control tab " in the left nav bar-> navigate to the "Metricsdc" tab.
 
![image](https://user-images.githubusercontent.com/54656883/66652851-52ad2e00-ec54-11e9-8a35-eb1710b0fe8a.png)

 
**Expected:**
When screen reader users navigate to the  "SQL Servers ","HDFS", "Sparks", "Control", "Gateway" and "App" controls, they should be announced with proper role as tab list , tab, and tab panel and even the selection of the currently selected tab should be announced to the users.

**User Impact:**
If proper role and proper structure of an interactive element is not announced to the screen reader users then they will get confused and might not know how to interact with which can lead to miss out some important information.

**Attachment for Reference:**
![7187_A11y_AzureDataStudio_BigDataClusterOverviewPage_ScreenReader](https://user-images.githubusercontent.com/54656883/67078106-42de9e00-f1ae-11e9-868d-4c67520a6a14.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
Fixed by https://github.com/microsoft/azuredatastudio/pull/8405

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_Mac;

**Comment by Rama8345** [comment]
**Github Tags: #A11ySev2;**

**Comment by manaswita-chougule** [comment]
#Regressed:07-18-23; #PendingInput;
@Charles-Gagnon 
Could you please help us with the details to get connected with "SQL Server Big Data Clusters" to verify the bug. Thanks!

**Comment by Charles-Gagnon** [comment]
This feature has been removed, so you can close/remove all Big Data Cluster issues.

**Comment by 12-shweta** [comment]
@Charles-Gagnon 
We will need to verify whether the feature is removed, then only we can close this bug.
So that we need server details to get connected and verify the bug.

**Comment by 12-shweta** [comment]
[like]  Shweta Warhadkar (Tata Consult... reacted to your message:
________________________________

**Comment by Charles-Gagnon** [comment]
@12-shweta The UI to enter in the server information doesn't exist anymore, so even if you had it there isn't anything you could do with it. And because of this the server was decommissioned. Let me know if this is a concern.

**Comment by 12-shweta** [comment]
As per confirmation above, closing this bug.
